styleguide/gulpfile.js

```diff
@@ -216,12 +210,6 @@ gulp.task('watch', function () {
     ['sass']
   );
 
-  // // Watch icons
```

Comment:
Since my work on this was WIP, I was actually planning on keeping this around. Just needed to comment it out to get the build working initially, and I have a to-do list item to put it back in.